src/main/java/backend/Model.java

```diff
@@ -102,7 +107,7 @@ public LR_Model(Instances d, String[] params) throws Exception {
     protected void run() throws Exception {
         //invoke crossfold validation (Classifier obj, Instance, #folds, RNG)
         eval.crossValidateModel(classifier, data, 10, new Random(1));
-        //System.out.println(eval.toSummaryString("\nResults\n======\n", false));
+        log(eval.toSummaryString("\nResults\n======\n", false));
```

Comment:
Created a summarize() method in base Model that performs the operation. I am pretty sure this will be fine for other classifiers unless they do not support summaries (possible but unlikely).